### PR TITLE
Vault page master accounts

### PR DESCRIPTION
**Pull request by despablos**

### CODE REVIEW

File: src/components/common/Details/DetailApp.js [hunk 1]
```diff
@@ -64,15 +63,19 @@ const DetailApp = (props: Props) => {
...
+              0x012345567abcdef012345567abcdef012345567abcdef012345567abc
+              <CopyButton value="0x012345567abcdef012345567abcdef012345567abcdef012345567abc" />
+            </a>
+          </span>
         </li>
         <li className="item">
           <span className="item-name">Master Accounts</span>
-          <span className="item-value">2 SMH</span>
+          <span className="item-value">
+            <a>
```
Comment (Motzart) [inline src/components/common/Details/DetailApp.js:75]: wrong tag a - see above, also add e.preventDefault() onClick method for stoping propagation, all links in this page 
shouldn't work, it's just mock data

File: src/helper/formatter.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export const fullDate = (unixTimestamp) => {
 // Format (00/00/0000)
 export const formattedDate = (unixTimestamp) => {
   const time = new Date(convertTime(unixTimestamp));
-  return `${time.getUTCMonth()}/${time.getUTCDate()}/${time.getUTCFullYear()}`
+  return `${time.getUTCMonth() + 1}/${time.getUTCDate()}/${time.getUTCFullYear()}`
```
Comment (Motzart) [inline src/helper/formatter.js:15]: add comments why it's needed

File: src/components/common/Details/DetailApp.js (hunk 1 above)
Comment (despablos) [reply src/components/common/Details/DetailApp.js]: done

File: src/helper/formatter.js (hunk 2 above)
Comment (despablos) [reply src/helper/formatter.js]: done

**Pull request merged**